security/contrib/debug_toolbar_log/templates/debug_toolbar/panels/sql.html

```diff
@@ -0,0 +1,117 @@
+{% load i18n l10n %}{% load static %}
```

Comment:
some debug toolbar have url links, but we didn't want it

Reply:
I don't see any usage of this file... will debug toolbar automatically use it? Aren't you overriding also the normal toolbar template?

Reply:
yes exactly.